tox.ini

```diff
@@ -48,6 +49,7 @@ setenv =
   hdp: HADOOP_DISTRO=hdp
   hdp: HADOOP_HOME={toxinidir}/.tox/hadoop-hdp
   contrib: NOSE_ATTR=contrib
+  snowflake: NOSE_ATTR=snowflake
```

Comment:
if snowflake is assed as its own tox type, we'll also want to `and not snowflake` from `core` (below)